network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,79 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-metrics-dashboard-enhancements"]
+==== Metrics, dashboards, and alerts enhancements
+The Network Observability metrics dashboards in *Observe* → *Dashboards* → *NetObserv* has new metrics types you can use to create Prometheus alerts. The list items are now in the `includeList` whereas before they were in `ignoreTags`. For a complete list of these metrics, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-improved-lokistack-integration"]
+==== Improvements for Network Observability without Loki
+You can create Prometheus alerts for the *Netobserv* dashboard using DNS, Packet drop, and RTT metrics, even if you don't use Loki. In 1.5, these metrics were only available for querying and analysis in the *Network Traffic*, *Overview*, and *Topology* views, which are not available without Loki. For more information, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-zones"]
+==== Availability zones
+You can configure the `FlowCollector` to collect information about the cluster availability zones. This allows you to enrich network flow data with the link:https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone[`topology.kubernetes.io/zone`] label value applied to the nodes. For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-zonesnw-observe-network-traffic[Working with availability zones].
+
+[id="network-observability-enhanced-configuration-and-ui-1.5"]
+==== Notable enhancements 
+
+[discrete]
+[id="web-console-enhancements-1.5_{context}"]
+===== Web console enhancements:
+
+* New panels added to the *Overview* view for DNS & RTT: Min, Max, P90, P99.
+* New panel display options to focus on one panel while keeping others viewable but smaller focus, switch graph type, and show *Top* and *Overall*.
+* Collection latency warning in the *Custom time range* pop-up window.
+* Enhanced visibility for *Manage panels* and *Manage columns* pop-up windows contents.
+* The Differentiated Services Code Point (DSCP) field for egress QoS is now available for filtering QoS DSCP in the web console *Network Traffic* page.
```

Comment:
@memodi as discussed on Slack, I added this bullet back. DSCP is only a release note mention. We do not have any other product documentation for using DSCP, as it is my understanding based on conversation with Amogh that DSCP is not something that needs to be configured by Network Observability customers. Its just a field that is now available by default for filtering in the Network Traffic table, which we don't document the complete list of those fields in the Network Traffic table. Please let me know if I'm wrong about any of this 😅

Reply:
that's right. No additional configuration is required in flowcollector on customers side. Thanks.